code/modules/atmospherics/machinery/pipes/heat_exchange/manifold.dm

```diff
@@ -27,7 +27,8 @@
 	//Add non-broken pieces
 	for(var/i in 1 to device_type)
 		if(nodes[i])
-			. += get_pipe_image(icon, "pipe-[piping_layer]", get_dir(src, nodes[i]))
+			var/turf/adjacent_node_loc = nodes[i].associated_loc
```

Comment:
that works?

Reply:
yes because nodes is typed as var/list/obj/machinery/atmospherics/nodes